resources/views/test-unit-notification.blade.php

```diff
@@ -0,0 +1,24 @@
+/* @@covers \{{ $covered }} */
+
+use {{ $classFqcn }};
+use Illuminate\Notifications\Messages\MailMessage;
+
+it('uses the mail channel', function (): void {
+    $notification = new {{ $classBasename }}();
+
+    expect($notification->via())->toContain('mail');
```

Comment:
```suggestion
    expect($notification->via())->toBe(['mail']);
```

Reply:
Done in f765190a76e70008ddcf5b00724935b2c5825fd2